styles/style.css

```diff
@@ -8,3 +8,90 @@ body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
 }
+
+#blue {
+  display: flex;
+  align-items: center;
+  background: linear-gradient(#3292e0, #1c5380);
+  padding-top: 10vw;
+  padding-bottom: 10vw;
+  padding-left: 50vw;
```

Comment:
A tip here, if you are defining all four paddings (or margins), you can do it in one line. I saw you used it already in some cases.

```
padding: 10vw 0 10vw 50vw; 
```

you would read this line: 10vw top, 0 right, 10vw bottom, 50vw left